modules/agar/src/main/ts/ephox/agar/api/TestStore.ts

```diff
@@ -55,6 +58,8 @@ export const TestStore = (): TestStore => {
 
   const assertEq = (label: string, expected: any[]) => Assert.eq(label, expected, array.slice(0));
 
+  const assertSortedEq = (label: string, expected: any[]) => Assert.eq(label, expected.slice(0).sort(), array.slice(0).sort());
+
   const sAssertSortedEq = (label: string, expected: any[]) => Step.sync(() =>
   // Can't use a normal step here, because we don't need to get array lazily
```

Comment:
We should probably get rid of that comment. I can't work out what it is intended to mean. I think there's a typo in there somewhere.

Reply:
Removed the comment.